app/views/courses/edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Editing Course</h1>
```

Comment:
I know this is boilerplate but we are 100% not going to be keeping this so I would just remove this entire file.